### PR TITLE
tier1-deployment-job: update job configuration

### PR DESCRIPTION
**Pull request by sunilkumarn417**
Signed-off-by: sunilkumarn417 <sunnagar@redhat.com>

- jenkins job configuration for tier-1 deployment test suites 

https://storage-jenkins-csb-ceph.cloud.paas.psi.redhat.com/view/RHCS%205%20QE/job/rhceph-5-tier-1-test/6/console

### CODE REVIEW

**Comment by sunilkumarn417** [comment]
**Latest Result:**
https://storage-jenkins-csb-ceph.cloud.paas.psi.redhat.com/view/RHCS%205%20QE/job/rhceph-5-tier-1-test-deploy/7/console (PASSED)

**Comment by sunilkumarn417** [comment]
**test result:**
https://storage-jenkins-csb-ceph.cloud.paas.psi.redhat.com/view/RHCS%205%20QE/job/rhceph-5-tier-1-test-deploy/3/

File: pipeline/5/Jenkinsfile-deployment-tier-1.groovy [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+    }
+
+    timeout(unit: "MINUTES", time: 180) {
+        parallel testStages0
+    }
+
+    timeout(unit: "MINUTES", time: 120) {
+        parallel testStages1
+    }
+
```
Comment (psathyan) [inline pipeline/5/Jenkinsfile-deployment-tier-1.groovy:153]: Please include a stage to send email. That is
```
    stage('Publish Results') {
        script {
            sharedLib.sendEMail("Deploy-Tier-1", test_results)
        }
    }
```

Comment (sunilkumarn417) [reply pipeline/5/Jenkinsfile-deployment-tier-1.groovy:153]: Hi @psathyan,
As discussed in last meeting, we planned not to include publish section. So RGW jenkins config went in without email section. https://github.com/red-hat-storage/cephci/blob/master/pipeline/5/jenkinsfile-object-tier-1.groovy

Let me know if we have to include this again.

Comment (sunilkumarn417) [reply pipeline/5/Jenkinsfile-deployment-tier-1.groovy:153]: Updated.

**Review by psathyan — APPROVED** [review]

**Pull request merged**